spec/interjectable_spec.rb

```diff
@@ -96,7 +103,7 @@
   end
 
   context "when included" do
-    let(:klass) { Class.new { extend Interjectable } }
+    let(:klass) { Class.new { include Interjectable } }
```

Comment:
LOL great catch

Reply:
Can we also add an assertion of like `klass.instance_method(:inject)` is nil?

Reply:
I added tests that try calling the non-existent methods.

Reply:
Mind if I merge?

Reply:
🚢 it! 👍